Public/Src/Cache/ContentStore/Test/Stores/FileSystemContentStoreInternalPutFileTests.cs

```diff
@@ -110,6 +112,61 @@ public Task PutFilePins()
             });
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public Task PutSameContentManyTimesTest(bool useRedundantPutFileShortcut)

```

Comment:
How long does it take to run this test? Just curious, because in some cases stress tests are taking relatively long time.

Reply:
A few seconds.